### DISCUSSION TITLE
How to use SharedPreferences?

### DISCUSSION THREAD

**Discussion by elibroftw** [unoplatform/uno#11481] [Q&A] [answered]
I want to save a JWT to SharedPreferences for my mobile application. How would I go about doing that?

I have come across https://platform.uno/docs/articles/features/file-management.html#support-for-cachedfilemanager

Here is how I did it 3 years ago in Flutter:
https://github.com/elibroftw/cloud-copy/blob/master/Cloud%20Copy%20Mobile/lib/main.dart#LL19-L19C21

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! [Application.Current.LocalSettings ](https://platform.uno/docs/articles/features/settings.html) is the way to store data on all platforms.

**Reply by elibroftw** [reply]
Helpful, thanks.
